base/file.jl

```diff
@@ -80,6 +104,18 @@ end
     cd(f::Function, dir::AbstractString=homedir())
 
 Temporarily changes the current working directory and applies function `f` before returning.
+
+# Examples
+```jldoctest
+julia> cd(readdir, joinpath(Sys.BINDIR, Base.DATAROOTDIR, "julia"))
```

Comment:
Perhaps overly complicated to use `Sys.BINDIR` and `Base.DATAROOTDIR` just to show how to use `cd`? It takes focus on what we are trying to show.

Reply:
Yeah, I wanted something that would work as a doctest but maybe a `julia-repl` is ok for now